tests/config/fgcmFitCycleHsc.py

```diff
@@ -1,7 +1,10 @@
 import lsst.fgcmcal as fgcmcal
 
 config.outfileBase = 'TestFgcm'
-config.physicalFilterMap = {'HSC-G': 'g', 'HSC-R': 'r', 'HSC-I': 'i'}
+config.physicalFilterMap = {'HSC-G': 'g',
+                            'HSC-R': 'r',
+                            'HSC-I': 'i',
+                            'HSC-Z': 'z'}
```

Comment:
Why add the `z` band mapping here if you don't use it (that I can see) in the tests?

Reply:
I've added a specific note that this is here to test the failure that is being fixed (namely, that if there were extra filters defined in the `physicalFilterMap` then `fgcm` would complain.)